### PR TITLE
Add HTCondor

### PR DESCRIPTION
**Pull request by sebastian-luna-valero**
To be added after https://github.com/grycap/ansible-role-htcondor/pull/10 is merged

### CODE REVIEW

File: tosca-templates/htcondor_cluster.yml [hunk 1]
```diff
@@ -78,7 +78,7 @@ topology_template:
       type: tosca.nodes.indigo.LRMS.FrontEnd.HTCondor
       properties:
         wn_ips: { get_attribute: [ lrms_wn, private_address ] }
-        htcondor_password: { get_input: htcondor_password }C
+        htcondor_password: { get_input: htcondor_password }
```
Comment (sebastian-luna-valero) [inline tosca-templates/htcondor_cluster.yml:81]: Interesting.

I am testing this in IM-dev, and I get:

```bash
TASK [grycap.htcondor : Store Condor credentials] ******************************
Wednesday 06 September 2023  11:56:32.961011 
fatal: [192.168.13.48_3]: FAILED! =&gt; {"msg": "An unhandled exception occurred while templating '{{ htcondor_password }}'. Error was a <class 'ansible.errors.ansibleerror'="">, original message: An unhandled exception occurred while templating '{{ htcondor_password }}'.
```
However, the ansible role works well when I use it out of IM.

Do you know why?

Comment (sebastian-luna-valero) [reply tosca-templates/htcondor_cluster.yml:81]: oh wait, should I wait for `grycap.htcondor` to be updated in https://galaxy.ansible.com/grycap/htcondor, right?

Comment (sebastian-luna-valero) [reply tosca-templates/htcondor_cluster.yml:81]: Interesting... 

How come that you don't get the ` "An unhandled exception occurred while templating '{{ htcondor_password }}'` error?

Comment (sebastian-luna-valero) [reply tosca-templates/htcondor_cluster.yml:81]: I did another test. I think the `htcondor_password` issue comes when you change the default password in the IM-dev web interface, under the `HTCondor configuration` tab, before clicking `Submit`?

Comment (micafer) [reply tosca-templates/htcondor_cluster.yml:81]: > I did another test. I think the `htcondor_password` issue comes when you change the default password in the IM-dev web interface, under the `HTCondor configuration` tab, before clicking `Submit`?

May be, as in my test I did not change the default value.
Let me check.

Comment (micafer) [reply tosca-templates/htcondor_cluster.yml:81]: I tested changing the password and it works.
Maybe you are using an invalid character.

File: tosca-templates/htcondor_cluster.yml [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+      default: 1 GB
+      required: yes
+    wn_cpus:
+      type: integer
+      description: Number of CPUs for the WNs
+      default: 1
+      required: yes
+    wn_mem:
+      type: scalar-unit.size
+      description: Amount of Memory for the WNs
```
Comment (sebastian-luna-valero) [inline tosca-templates/htcondor_cluster.yml:38]:
```suggestion
      description: Amount of Memory for the Worker Nodes
```

File: tosca-templates/htcondor_cluster.yml [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+      default: 1 GB
+      required: yes
```
Comment (sebastian-luna-valero) [inline tosca-templates/htcondor_cluster.yml:40]:
```suggestion
      required: yes
      constraints:
      - valid_values: [ 2 GB, 4 GB, 8 GB, 16 GB, 32 GB, 64 GB, 128 GB, 256 GB, 512 GB ]
```

**Review by sebastian-luna-valero — COMMENTED** [review]
I would like to add these changes as well, as they are more intuitive for the end user.

**Pull request merged**